portfolio/src/main/webapp/style.css

```diff
@@ -40,13 +51,63 @@ cursive {
 }
 
 div.gallery img {
-  height: auto;
+  height: 75%;
   width: 100%;
 }
 
 div.desc {
+  background-color: white;
+  font-family: cursive;
+  font-size: 14px;
+  margin-top: -4px;
+  padding: 13px;
+  text-align: center;
+  width: 177px;
+}
+
+media-title {
   font-family: cursive;
-  font-size: 12px;
-  padding: 2px;
+  margin-top: 10px;
+} 
+
+.fa {
+  border-radius: 50%;
+  cursor: pointer;
+  font-size: 30px;
+  margin-bottom:150px;
+  margin-right: 12px;
+  padding: 20px;
   text-align: center;
+  text-decoration: none;
+  width: 15px;
+}
+
+.fa:hover {
+  opacity: 0.7;
+}
+
+.fa-linkedin {
+  background: #007bb5;
+  color: white;
```

Comment:
Can you add the `color: white;` to `.fa` instead, since all of these seem to use the same value?